src/partials/home/companies.js

```diff
@@ -12,74 +9,69 @@ import AIRBNB from "../../../static/logos/logo-airbnb.svg";
+import {
+  LinkButton,
+  LandingSectionWrapper,
+  LandingSectionContent
+} from "./styles";
 
-class Companies extends React.Component {
-  render() {
-    return (
-      <div className="Companies">
-        <h2 className="SubHeading">A few of our fans</h2>
-        <ul className="Companies-List">
-          <img
-            className="Companies-logo"
-            style={{ maxWidth: "171px", filter: "invert(1)" }}
-            src={AIRBNB}
-          />
-          <img
-            className="Companies-logo"
-            style={{ maxWidth: "300px", filter: "invert(1)" }}
-            src={FIVETHIRTYEIGHT}
-          />
-          <img
-            className="Companies-logo"
-            style={{
-              maxWidth: "177px",
-              paddingTop: "2px"
-            }}
-            src={REDFIN}
-          />
-          <img
-            className="Companies-logo"
-            style={{ maxWidth: "350px" }}
-            src={USAFACTS}
-          />
-        </ul>
-        <ul className="Companies-List">
-          <img
-            className="Companies-logo"
-            style={{ maxWidth: "200px", filter: "invert(1)" }}
-            src={VIACOM}
-          />
-          <img
-            className="Companies-logo"
-            style={{ maxWidth: "300px", height: "50px" }}
-            src={BENAROYA}
-          />
-          <img
-            className="Companies-logo"
-            style={{
-              maxWidth: "190px",
-              paddingBottom: "3px",
-              filter: "invert(1)"
-            }}
-            src={ZILLOW}
-          />
-          <img
-            className="Companies-logo"
-            style={{ maxWidth: "141px", filter: "invert(1)" }}
-            src={TUNE}
-          />
-        </ul>
-        <p className="SubHeading">
-          <Link to="/about#showcase" className="isLight">
-            See showcase&nbsp;
-            <Icon glyph="internal-link" style={{ filter: "invert(1)" }} />
-          </Link>
-        </p>
-      </div>
-    );
+const SectionHeading = styled.h2`
+  color: ${({ theme }) => theme.color.darkBrown};
+  font-size: 18px;
+  font-weight: bold;
+  line-height: 1.58;
+  letter-spacing: 0.48px;
+  margin: 0;
+  text-align: center;
+  @media ${({ theme }) => theme.mediaQuery.md} {
+    font-size: 24px;
   }
-}
+`;
+
+const CompaniesList = styled.ul`
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-rows: auto;
+  grid-gap: 4rem;
+  margin: 5rem 0;
+  @media ${({ theme }) => theme.mediaQuery.sm} {
+    grid-template-columns: repeat(4, 1fr);
+    grid-auto-rows: 1fr;
+    grid-gap: 3rem;
+  }
+`;
+
+const CompanyLogo = styled.img`
+  align-self: center;
+  justify-self: center;
+  max-height: 100px;
+  max-width: 200px;
+  width: 100%;
+`;
+
+const Companies = () => (
+  <LandingSectionWrapper bg={importedTheme.color.lightGray}>
+    <LandingSectionContent>
+      <SectionHeading>A Few of Our Fans</SectionHeading>
+      <CompaniesList>
```

Comment:
Nit: this `ul` is expecting `li`s and is getting `img`s, maybe wrap the `img` in `li` so that the structure remains.